source/scripting_v3/GTA/Entities/Peds/Ped.cs

```diff
@@ -2956,6 +2956,27 @@ public void ResetWeaponMovementClipSet()
         public PedMoveNetworkTaskInterface MoveNetworkTaskInterface
             => _moveNetworkInterface ?? (_moveNetworkInterface = new PedMoveNetworkTaskInterface(this));
 
+        /// <summary>
+        /// Sets a facial animation override for this <see cref="Ped"/>.
+        /// </summary>
+        /// <param name="facialAnimation">
+        /// The facial animation to set as override.
+        /// </param>
+        /// <remarks>
+        /// Not every facial animation works for every <see cref="Ped"/> model/gender.
+        /// </remarks>
+        public void SetFacialAnimationOverride(PedFacialAnimation facialAnimation)
+            => Function.Call(Hash.SET_FACIAL_IDLE_ANIM_OVERRIDE, Handle, facialAnimation.GetAnimationName(), 0);
```

Comment:
Yes, passing 0 would work as `SET_FACIAL_IDLE_ANIM_OVERRIDE` don't try to dereference when the 3rd argument is set to 0. However, specifying `null` will be better because that tells readers that the 3rd argument isn't supposed to be numeric values.